signinwithapplebutton/src/main/java/com/willowtreeapps/signinwithapplebutton/view/SignInWebViewDialogFragment.kt

```diff
@@ -64,7 +65,7 @@ internal class SignInWebViewDialogFragment : DialogFragment(), SignInWithAppleCa
         }
 
         webView.webViewClient = authenticationAttempt?.let {
```

Comment:
Now that we're packaging authenticationAttempt with the fragment arguments, do you think it makes sense to treat it as not nullable? Is `lateinit` the right construct for that?

Reply:
Yeah makes sense, think I'll create another pr since that's more of a follow-up on https://github.com/willowtreeapps/sign-in-with-apple-button-android/pull/37 than this.